prac_06/guitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    print("My guitars!")
+    guitars = []
+    name = input("Name: ")
+    while name:
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitars.append(Guitar(name, year, cost))
+        print(f"{name} ({year}) : ${cost:.2f} added.\n")
+        name = input("Name: ")
+    print("These are my guitars:")
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = " (vintage)" if guitar.is_vintage() else ""
```

Comment:
I really like the line here with the if statement in the same line, makes for a simple and neat program

Reply:
Thanks